### PR TITLE
fixes #5: Add new flag of UseLastRowAsFooter

### LINKED ISSUE #5: Add flag to specify the last row to be a footer
There is no way to indicate whether the last row of the table should be considered a footer row (in the context of `thead` / `tbody` / `tfoot` ) if desired

### PR DESCRIPTION
**Pull request by Nysosis**
fixes #5 

Doesn't do anything within the context of the table data itself, just a flag so when rendering you can handle it how you want

### CODE REVIEW

File: src/Limbo.Umbraco.Tables/wwwroot/Lang/da-DK.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 		<key alias="addContent">Tilføj indhold</key>
 		<key alias="useFirstRowAsHeader">Brug første række som header</key>
 		<key alias="useFirstColumnAsHeader">Brug første kolonne som header</key>
+		<key alias="useLastRowAsFooter">Brug sidste række som sidefod</key>
```
Comment (Nysosis) [inline src/Limbo.Umbraco.Tables/wwwroot/Lang/da-DK.xml:10]: This was a Google translate job, as I don't know Danish

**Comment by abjerner** [comment]
Hi @Nysosis and thanks for this PR 👍 

I guess there are use cases where this feature makes sense, but I could also see situations where this doesn't make sense. Eg. if the frontend doesn't support a table footer - or users shouldn't be able to the table footer in the backoffice.

So maybe there needs to data type option to enables or disables this option in the property. I'll need to have a chat about this with my colleagues. I'll report back with my findings 😉

File: src/Limbo.Umbraco.Tables/wwwroot/Lang/da-DK.xml (hunk 1 above)
Comment (abjerner) [reply src/Limbo.Umbraco.Tables/wwwroot/Lang/da-DK.xml:10]: `sidefod` equals `page footer`, so it's not entirely correct. It should probably be either `tabelfod` (table footer) or simply `footer` as we're also using `header` in English to describe the header.

**Comment by abjerner** [comment]
Hi @Nysosis 

We've decided to merge this PR, but like I mentioned in my previous comment, we think there should be a data type option that controls whether this flag is enabled in the property. I will look into adding this 😉 

Thanks for submitting this PR 👍

**Pull request merged**